index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content= "width=device-width, initial-scale=1.0">
+        <meta content="text" charset="utf-8">
+        <link rel="stylesheet" href="styles.css">
+        <title>Kiplaacollo Portfolio website</title>
+
+    </head>
+    <body>
+        <header class="head">
+                <h3 class="logo">Welcome</h3>
+                <img src="images/Enabled.png" alt="menu icon">
+        </header>
+        <div class="content">
+                <div class="primary_txt">
+                    <h2 class="top_color">Hey there. I'm Kiplaacollo </h2>
+                    <h2 class="bottom_color">I'm a software developer </h2>
```

Comment:
- [ ] Kindly I recommend you to use `h1` tag for main title and `h2` tag for sub-title , This way will be semantically correct and the screen readers and other technologies will thank you. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.